Seda/Seda/AppDelegate.swift

```diff
@@ -21,7 +22,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         IQKeyboardManager.shared.enable = true
         IQKeyboardManager.shared.shouldResignOnTouchOutside = true
         IQKeyboardManager.shared.enableAutoToolbar = false
-        ScanViewController.configure(apiKey: "fkeuabRTa4p3x6sNrLtv0JPDaNiEDIv1")
+        ScanViewController.configure(apiKey: "yTAstrsjCkEQ45aoUXNnGdDqd1JNXBXQ")
```

Comment:
What is this apiKey for?

Reply:
the api key for the CardScan. I created using the link they put on https://github.com/getbouncer/cardscan-ios. and the link is https://api.getbouncer.com/console

Reply:
its for CardScan. Not sure why it changed though, I'm guessing both of us made accounts and we used different keys.

Reply:
yes. I met an error when I built it so I changed it to a new key.